keps/sig-storage/361-local-ephemeral-storage-isolation/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title:  Local Ephemeral Storage Capacity Isolation 
+kep-number: 361
+authors:
+  - "@jinxu97"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+status: beta
```

Comment:
"implementable", and after all PRs are in or after release, it gets updated to "implemented".